cli/src/pcluster/resources/imagebuilder/parallelcluster.yaml

```diff
@@ -197,7 +197,20 @@ phases:
           commands:
             - |
               set -v
+              PLATFORM='{{ build.PlatformName.outputs.stdout }}'
+
+              if [[ ${!PLATFORM} == RHEL ]]; then
+                CA_CERTS_FILE=/etc/ssl/certs/ca-bundle.crt
+              elif [[ ${!PLATFORM} == DEBIAN ]]; then
+                CA_CERTS_FILE=/etc/ssl/certs/ca-certificates.crt
```

Comment:
Please add:
```
apt upgrade ca-certificates
```

Reply:
I went with `apt-get` for consistency with the remaining of the file, such as in `InstallPrerequisite`.

According to the man page, `upgrade` will upgrade all packages. I think the preferred way to upgrade a single package is with `install` and `--only-upgrade`.